tests/SCF/test_SCF.py

```diff
@@ -1,19 +1,88 @@
 import pytest
 import SCF
+import pickle
+from mol import mol
+
+
+Suv = pickle.load(open("suv.pkl", "rb"))

```

Comment:
These should not be in here, you should use the functions in the fixture to do this assignment. Travis if failing because you did not checkin the real_tuv.pkl file.